CRM/Case/BAO/Case.php

```diff
@@ -1212,9 +1212,20 @@ public static function getCaseActivity($caseID, &$params, $contactID, $context =
    *
    */
   public static function getRelatedContacts($caseID, $skipDetails = FALSE) {
```

Comment:
yep, from a coding POV I think getting additional data should be opt in, not opt out. Reading "IF NOT SKIP DETAIL" is brain hurty & means the same thing as "IF INCLUDE CASEROLEDETAILS" which seems much more readable.

 I guess you could argue the api should opt in to getting roles, but there is a third place in the code that calls this function.